AmplifyPlugins/Internal/Sources/InternalAWSPinpoint/Endpoint/EndpointInformation.swift

```diff
@@ -8,26 +8,24 @@
 import Amplify
 import Foundation
 
-protocol EndpointInformation {
+struct EndpointInformation {
```

Comment:
why this change? or is it because of rebase?

Reply:
the original intent was to have `DeviceInfo` implement the `EndpointProtocol`, this simplifies it by having the provider return the struct that is constructed from `DeviceInfo`